ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/spanning-tree.j2

```diff
@@ -2,14 +2,69 @@
 {% if spanning_tree is defined and spanning_tree is not none %}
 ## Spanning Tree Summary
 
-Mode: {{ spanning_tree.mode }}
+STP mode: **{{ spanning_tree.mode }}**
 {%     if spanning_tree.mode == "mstp" %}
+{%         if spanning_tree.mst is defined and spanning_tree.mst is not none %}
+{%             if spanning_tree.mst.pvst_border is defined and spanning_tree.mst.pvst_border == true %}
+MST PSVT Border is enabled.
+{%             endif %}
+{%         endif %}
 
```

Comment:
It is not super important, but this section with pvst_border should really be under "General Spanning-Tree Settings" or?